ecs_windows_forward_daemon/README.md

```diff
@@ -0,0 +1,234 @@
+# Default configurations for Windows images in Amazon ECS
+## Using Fluent Bit on Amazon ECS Windows in a Daemon configuration using forward input plugin
+
+In order to allow Amazon ECS Windows customers to get started promptly with Fluent Bit without performing significant configuration, we have crafted default configurations for the following output destinations-
+* [Amazon CloudWatch](https://github.com/aws/aws-for-fluent-bit/blob/mainline/ecs_windows_forward_daemon/cloudwatch.conf)
+* [Amazon S3](https://github.com/aws/aws-for-fluent-bit/blob/mainline/ecs_windows_forward_daemon/s3.conf)
+* [Amazon Kinesis Data Streams](https://github.com/aws/aws-for-fluent-bit/blob/mainline/ecs_windows_forward_daemon/kinesis.conf)
+* [Amazon Kinesis Data Firehose](https://github.com/aws/aws-for-fluent-bit/blob/mainline/ecs_windows_forward_daemon/firehose.conf)
+
+These default configurations are baked in the Windows container images with image versions 2.28.4 and above. These can be used in a Daemon configuration which uses forward input plugin.
```

Comment:
nit, I think this is slightly unclear: "in a Daemon Service deployment with a configuration that uses the forward.."